spec/integration/claims/aud_spec.cr

```diff
@@ -5,7 +5,7 @@ describe "aud claim" do
     context "aud options is not passed to .decode method" do
       it "accepts token" do
         token = JWT.encode({"foo" => "bar"}, "key", "HS256")
-        payload, header = JWT.decode(token, "key", "HS256")
+        payload, _ = JWT.decode(token, "key", "HS256")
```

Comment:
Just a quick question, why using `_`? Should it increase performance or?

Reply:
Also it should ideally be splitted into another commit, just saying.

Reply:
Ok